fhir-persistence/src/main/java/com/ibm/fhir/persistence/payload/FHIRPayloadPersistence.java

```diff
@@ -25,40 +24,34 @@
      * @param resourceTypeId the database id assigned to this resource type
```

Comment:
```suggestion
     * @param resourceTypeName
     * @param resourceTypeId the database id assigned to this resource type
```

Reply:
The signature includes both resourceTypeName and resourceTypeId.